### PR TITLE
[bitnami/contour] Synchronized all CRD with official yaml render v0.2.9 > v0.5.0

### PR DESCRIPTION
**Pull request by VA-PLE**
Update httpproxies.projectcontour.io to v0.5.0

### CODE REVIEW

**Review by Mauraza — CHANGES_REQUESTED** [review]
Hi @VA-PLE,

Thanks for the PR! 
Could you update the version of the chart and try to keep the descriptions they were (the longitude of the line)?

**Comment by VA-PLE** [comment]
> Hi @VA-PLE,
> 
> Thanks for the PR!
> Could you update the version of the chart and try to keep the descriptions they were (the longitude of the line)?

Yes, but i used data from official contour.yaml https://raw.githubusercontent.com/projectcontour/contour/release-1.17/examples/render/contour.yaml

**Comment by Mauraza** [comment]
Hi @VA-PLE,

the length of the line is a style parameter, the content will be the same. Do not forget the chart update version in Chart.yaml

**Comment by VA-PLE** [comment]
@Mauraza This is the official yaml render 
https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml
I think it's better to sync them with the official one, for easier support. If you look more closely you will see that in addition to the descriptions, new fields have been added in 0.5.0 version. I have no desire to edit more than 1000 lines of description

**Comment by marcosbc** [comment]
Thanks for the contribution @VA-PLE! I'm missing the chart version update, could you check that?.

In this case, since this is changing the CRD, I think we may need to increase the major version or chart upgrades will not work. In that case, it would also be good to add a note in the "Upgrading" section in the README describing this change, and how to upgrade. What do you think?

**Comment by VA-PLE** [comment]
@marcosbc I synchronized all CRD with official yaml render https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml and fixed missing the chart version update. I think that's enough.

File: bitnami/contour/Chart.yaml [hunk 1]
```diff
@@ -27,4 +27,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/bitnami-docker-contour
   - https://projectcontour.io
-version: 4.3.10
+version: 4.4.10
```
Comment (Mauraza) [inline bitnami/contour/Chart.yaml:30]: @marcosbc https://github.com/bitnami/charts/pull/6907#issuecomment-880763156 said that:

> I think we may need to increase the major version or chart upgrades will not work

Did you test the upgrade and works?

File: bitnami/contour/Chart.yaml [hunk 2]
```diff
@@ -27,4 +27,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/bitnami-docker-contour
   - https://projectcontour.io
-version: 4.3.11
+version: 4.4.11
```
Comment (Mauraza) [inline bitnami/contour/Chart.yaml:30]: Could you update to major (5.0.0)?

**Review by Mauraza — COMMENTED** [review]
Hi @VA-PLE,

Could you add a note in the "Upgrading" section in the README describing the change?

File: bitnami/contour/Chart.yaml (hunk 2 above)
Comment (VA-PLE) [reply bitnami/contour/Chart.yaml]: done

File: bitnami/contour/README.md [hunk 3]
```diff
@@ -401,6 +401,16 @@ Find more information about how to deal with common errors related to Bitnami’
 
 Please carefully read through the guide "Upgrading Contour" at https://projectcontour.io/resources/upgrading/.
 
+### To 5.0.0
+
+In this version it was synchronized CRD with the official Contour repository https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml
```
Comment (Mauraza) [inline bitnami/contour/README.md:406]: Could you change this Contour repository https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml to ⬇️?
```markdown
[Contour repository](https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml)
```
In that way, you can see that as a link: 
 [Contour repository](https://github.com/projectcontour/contour/blob/main/examples/render/contour.yaml)

**Review by Mauraza — COMMENTED** [review]
Hi @VA-PLE,

It's almost finished, one more change and we are ready to go.

File: bitnami/contour/README.md (hunk 3 above)
Comment (VA-PLE) [reply bitnami/contour/README.md]: done

**Review by Mauraza — APPROVED** [review]
Hi @VA-PLE,

Thanks for the contribution. LGTM!

**Pull request merged**